src/main/java/spark/globalstate/ServletFlag.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 - Per Wendel
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package spark.globalstate;
+
+/**
+ * Holds the global information if Spark was run from an "external" web application server.
+ */
+public class ServletFlag {
+
+    private static boolean isRunFromServlet = false;
```

Comment:
If this is truly "global state", should you use an AtomicBoolean here, to protect against possible thread access races?